res/controllers/Numark Mixtrack 2.midi.xml

```diff
@@ -481,7 +481,7 @@
             </control>
             <control>
                 <group>[Playlist]</group>
-                <key>NumarkMixTrackII.toggleDirectoryMode</key>
+                <key>NumarkMixTrackII.toggleSelectedSidebarItem</key>
```

Comment:
There is another `toggleDirectoryMode` binding above (0x77), is that the Back button?

That could use `[Library],MoveFocus`

Reply:
Idd, 0x77 is the back button, 0x76 is pressing the big browse button.